app.py

```diff
@@ -336,7 +336,21 @@ def parse():
 
 def invalid_space_error_to_response(ex, translations):
     warning = translate_error(ex.error_code, translations, vars(ex))
-    code = "# coding=utf8\n" + ex.fixed_code
+    if ex.has_turtle:
```

Comment:
This is now duplicated with lines 290 and further, maybe we can store this is a constant called TURTLE_PREFIX_CODE or something?

Reply:
Understood I'm changing it and will upload the changes!

Reply:
Thanks!!